src/service/http/interceptors.ts

```diff
@@ -5,13 +5,32 @@ import i18n from '../../utils/i18n'
 import { AppDispatch } from '../../contexts/reducer'
 import { AppActions } from '../../contexts/actions'
 
-const updateNetworkError = (dispatch: AppDispatch, occurError: boolean) => {
+let timeout: ReturnType<typeof setTimeout> | null
+
+const updateNetworkError = (dispatch: AppDispatch, occurError: boolean, errMessage = 'toast.invalid_network') => {
```

Comment:
Looks like the second parameter is always `true`

Reply:
I didn't notice that `false` has all deleted.
https://github.com/nervosnetwork/ckb-explorer-frontend/pull/980/commits/fa773f4407d9e83eb2940354edee7d93375b8d87